lib/sps_king/message.rb

```diff
@@ -21,22 +22,30 @@ class Message
     def initialize(account_options = {})
       @grouped_transactions = {}
       @account = account_class.new(account_options)
+      @schema_name = get_schema
```

Comment:
No schema should be defined when the message is first initialized, since that will be defined on the `to_xml` call.